Cargo.toml

```diff
@@ -385,7 +385,7 @@ pretty_assertions = "1.3.0"
 proc-macro2 = "1.0.79"
 qstring = "0.7.2"
 quote = "1.0.23"
-rand = "0.8.5"
+rand = "0.9.0"
```

Comment:
Updated the dependency, there were a couple callsites that needed minor updates.